core/authenticate/service.go

```diff
@@ -114,6 +121,9 @@ func (s Service) SupportedStrategies() []string {
 	if s.mailDialer != nil {
 		strategies = append(strategies, MailOTPAuthMethod.String(), MailLinkAuthMethod.String())
 	}
+	if s.config.PassKey.RPID != "" && len(s.config.PassKey.RPOrigins) != 0 {
```

Comment:
Instead of config, we can check if webauth config was initialized properly. `if s.webAuth != nil`

Reply:
done